snitun/client/connector.py

```diff
@@ -89,7 +103,10 @@ async def handler(
             # Process stream from multiplexer
             while not writer.transport.is_closing():
                 if not from_endpoint:
-                    from_endpoint = self._loop.create_task(reader.read(4096))
```

Comment:
Need to make sure we are pausing the right reader.  It should be the one that we are reading data from that will be delivered to the remote side input